### PR TITLE
Add session cookie deprecation notice

### PR DESCRIPTION
**Pull request by ghost**
https://issues.newrelic.com/browse/NR-99046
Follow-up to https://github.com/newrelic/docs-website/pull/12090/ .

### CODE REVIEW

File: src/content/docs/browser/new-relic-browser/page-load-timing-resources/instrumentation-browser-monitoring.mdx [hunk 1]
```diff
@@ -64,6 +64,9 @@ We use these methods to collect the page load timing information:
 
       <td>
         For browsers that do not implement the Navigation Timing Specification API, we rely on the [NREUM cookie](/docs/browser/new-relic-browser/miscellaneous/new-relic-cookies#nreum) and the browser agent to collect timing information.
+        <Callout variant="important">
+  For browser agent version 1217 and higher, usage of session cookies has been deprecated.
```
Comment (ally-sassman) [inline src/content/docs/browser/new-relic-browser/page-load-timing-resources/instrumentation-browser-monitoring.mdx:68]:
```suggestion
  For browser agent version 1220 and higher, [usage of session cookies has been deprecated](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1220/#removed-3rd-party-cookies), so this information is only relevant if you're using browser agent version lower 1216 and lower. 
```

**Review by metal-messiah — DISMISSED** [review]

File: src/content/docs/browser/browser-monitoring/page-load-timing-resources/new-relic-cookies-used-browser.mdx [hunk 2]
```diff
@@ -107,7 +105,7 @@ The default browser agent configuration does not retain any personal data as def
     title="JSESSIONID (deprecated)"
   >
 
-The JSESSIONID cookie is deprecated for agent version v1217 and higher. Session tracking is no longer done through cookies.
+The JSESSIONID cookie is deprecated for agent version 1220 and higher. Session tracking is no longer done through cookies.
```
Comment (ally-sassman) [inline src/content/docs/browser/browser-monitoring/page-load-timing-resources/new-relic-cookies-used-browser.mdx:108]:
```suggestion
The `JSESSIONID `cookie is deprecated for agent version 1220 and higher. Session tracking is no longer done through cookies.
```

File: src/content/docs/browser/browser-monitoring/page-load-timing-resources/new-relic-cookies-used-browser.mdx [hunk 3]
```diff
@@ -107,7 +105,7 @@ The default browser agent configuration does not retain any personal data as def
     title="JSESSIONID (deprecated)"
   >
 
-The JSESSIONID cookie is deprecated for agent version v1217 and higher. Session tracking is no longer done through cookies.
+The `JSESSIONID `cookie is deprecated for agent version 1220 and higher. Session tracking is no longer done through cookies.
```
Comment (ally-sassman) [inline src/content/docs/browser/browser-monitoring/page-load-timing-resources/new-relic-cookies-used-browser.mdx:108]:
```suggestion
The `JSESSIONID` cookie is deprecated for agent version 1220 and higher. Session tracking is no longer done through cookies.
```

**Review by rhetoric101 — APPROVED** [review]
@ally-sassman thanks for making these updates about session cookies!

**Pull request merged**